src/common/cups_print.c

```diff
@@ -115,9 +123,10 @@ static int _dest_cb(void *user_data, unsigned flags, cups_dest_t *dest)
   // check that the printer is ready
   if (psvalue!=NULL && strtol(psvalue, NULL, 10) < IPP_PRINTER_STOPPED)
   {
-    dt_printer_info_t *pr = dt_get_printer_info(dest->name);
-    if (pctl->cb) pctl->cb(pr, pctl->user_data);
-    free(pr);
+    dt_printer_info_t pr;
```

Comment:
You should init `pr`. Otherwise values like `is_turboprint` might be random. Or make sure that all values get set in `dt_get_printer_info()`.

Reply:
This got fixed by the subsequent memset. All good now.

Reply:
Great, so I'll merge. Thanks.